src/FsAutoComplete.Core/CommandResponse.fs

```diff
@@ -248,6 +264,11 @@ module CommandResponse =
 
   let info (serialize : Serializer) (s: string) = serialize { Kind = "info"; Data = s }
   let error (serialize : Serializer) (s: string) = serialize { Kind = "error"; Data = s }
```

Comment:
I don't think breaking contract is an issue, we've done it in the past, we'll just bump version on the release.

I think using `ResponseError<unit>` and some generic error code will provide more unified API. Also on the client instead of checking if `Code` field is defined people will be able to check if `Code = -1` (or whatever value) which may be easier to do.